actionlist/app/lib/commons.js

```diff
@@ -0,0 +1,22 @@
+
+export const preferenceJSON = "{\"pageSize\":10,\"refreshRate\":15,\"showPrimaryDelegations\":true,\"showSecondaryDelegations\":false,\"showSummary\":true,\"showFilters\":true,\"columns\":[{\"name\":\"processInstanceId\",\"title\":\"Process Instance\",\"link\":\"processInstance\"},{\"name\":\"processType\",\"title\":\"Process Type\"},{\"name\":\"title\",\"title\":\"Title\"},{\"name\":\"log\",\"title\":\"Log\",\"icon\":\"log\",\"link\":\"log\"}],\"colors\":{\"S\":\"red\",\"A\":\"orange\"},\"notifications\":{\"defaultFrequency\":\"immediate\",\"primaryDelegation\":true,\"secondaryDelegation\":false,\"approval\":true,\"complete\":true,\"acknowledge\":true,\"fyi\":false,\"processTypeOverrides\":{\"Some Process\":\"daily\"}}}";
+
+// Hack to initialize rowId and expanded state
+function manipulateList(list) {
+  const temp = list;
```

Comment:
You know this isn't creating a new object, right? You should be using the spread operator here and then doing an `Array.map` to add the `expanded` prop to each element.